### PR TITLE
14861-include-elevation-data-srtm90m-and-gmted250m

### PR DESCRIPTION
**Pull request by lgudyma-kontur**

### CODE REVIEW

File: scripts/build_ckan_dataset_description.py [hunk 1]
```diff
@@ -71,7 +71,10 @@ def build_ckan_dataset_description(strOutputFileName, number_of_objects, strFilt
         ckan_dataset_description["url"] = "https://hub.worldpop.org/"
         ckan_dataset_description["license_id"] = "cc-by"
     elif source == "srtm":
-        ckan_dataset_description["url"] = "https://www.earthdata.nasa.gov/"
+        ckan_dataset_description["url"] = "https://www.usgs.gov/"
+        ckan_dataset_description["license_id"] = "other-pd"
+    elif source == "gmted250":
```
Comment (Zkir) [inline scripts/build_ckan_dataset_description.py:76]: Is not this source **gmted2010** ?

**Review by Zkir — CHANGES_REQUESTED** [review]
it seems that the source  code for gmted is "gmted210".
it should be also added to ma_dictionaries.py file

File: scripts/mapaction_extract_country_from_srtm.sh [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
 # clip with country polygon
-gdalwarp -of GTIFF -cutline static_data/countries/$1 -crop_to_cutline -overwrite data/mid/srtm/srtm.vrt "${outdir}/${country_code}_elev_dem_ras_s0_srtm_pp_elevation30m.tif"
+if [ $filetype = "srtm30m" ]; then
+    gdalwarp -of GTIFF -cutline static_data/countries/$1 -crop_to_cutline -overwrite data/mid/srtm30m/srtm.vrt "${outdir}/${country_code}_elev_dem_ras_s0_srtm_pp_elevation30m.tif"
+ elif [ $filetype = "srtm90m" ]; then
+    gdalwarp -of GTIFF -cutline static_data/countries/$1 -crop_to_cutline -overwrite data/mid/srtm90m/srtm.vrt "${outdir}/${country_code}_elev_dem_ras_s0_srtm_pp_elevation90m.tif"
+ elif [ $filetype = "gmted250m" ]; then
+    # special naming for gmted250m datasource
+    gmted250m_inp=/vsizip/data/in/gmted250m/gmted250m.zip/be75_grd/sta.adf
+    gdalwarp -of GTIFF -cutline static_data/countries/$1 -crop_to_cutline -overwrite $gmted250m_inp "${outdir}/${country_code}_elev_dem_ras_s0_gmted2010_pp_elevation250m.tif"
```
Comment (Zkir) [inline scripts/mapaction_extract_country_from_srtm.sh:24]: Here you have gmted2010!

File: scripts/build_ckan_dataset_description.py (hunk 1 above)
Comment (Zkir) [reply scripts/build_ckan_dataset_description.py:76]: Compare it with line 24 of 
there we have gmted2010
`${outdir}/${country_code}_elev_dem_ras_s0_gmted2010_pp_elevation250m.tif"`

Source is that segment!

**Review by Zkir — CHANGES_REQUESTED** [review]

File: scripts/build_ckan_dataset_description.py (hunk 1 above)
Comment (lgudyma-kontur) [reply scripts/build_ckan_dataset_description.py]: i fixed the error now

**Review by Zkir — APPROVED** [review]
Ok, good for now.

**Pull request merged**